src/pages/manage/withdraw/withdraw.component.ts

```diff
@@ -103,7 +103,7 @@ export class WithdrawComponent {
    * @param res
    * @param displayAmount
    */
-  private handleWithdrawal(res, displayAmount) {
+  public handleWithdrawal(res, displayAmount) {
```

Comment:
The way we are testing Ionic requires us to instantiate the angular component. So we can't access helper method since the test is not within the class itself.